lib/libbol.py

```diff
@@ -6,6 +6,7 @@
 from collections import OrderedDict
 from io import BytesIO
 from copy import deepcopy
+from os.path import join, exists
```

Comment:
Usually we'd refrain from polluting the global namespace. It'd be best to simply import `os` (or `os.path` as a special case), but in no case importing `join` and `exists` alone. It's not justified in this case.